sql/moz-fx-data-shared-prod/monitoring_derived/jobs_by_organization_v1/schema.yaml

```diff
@@ -118,22 +118,22 @@ fields:
   type: RECORD
   fields:
   - mode: NULLABLE
-    name: elapsedMS
+    name: elapsed_ms
     type: STRING
   - mode: NULLABLE
-    name: totalSlotMs
+    name: total_slot_ms
     type: STRING
   - mode: NULLABLE
-    name: pendingUnits
+    name: pending_units
     type: STRING
   - mode: NULLABLE
-    name: completedUnits
+    name: completed_units
     type: STRING
   - mode: NULLABLE
-    name: activeUnits
+    name: active_units
     type: STRING
   - mode: NULLABLE
-    name: estimatedRunnableUnits
+    name: estimated_runnable_units
```

Comment:
I think you should translate this schema into YAML:

```json
{
  "fields": [
    {
      "name": "cache_hit",
      "type": "BOOLEAN",
      "mode": "NULLABLE"
    },
    {
      "name": "creation_time",
      "type": "TIMESTAMP",
      "mode": "NULLABLE"
    },
    {
      "name": "destination_table",
      "type": "RECORD",
      "mode": "NULLABLE",
      "fields": [
        {
          "name": "project_id",
          "type": "STRING",
          "mode": "NULLABLE"
        },
        {
          "name": "dataset_id",
          "type": "STRING",
          "mode": "NULLABLE"
        },
        {
          "name": "table_id",
          "type": "STRING",
          "mode": "NULLABLE"
        }
      ]
    },
    {
      "name": "end_time",
      "type": "TIMESTAMP",
      "mode": "NULLABLE"
    },
    {
      "name": "error_result",
      "type": "RECORD",
      "mode": "NULLABLE",
      "fields": [
        {
          "name": "reason",
          "type": "STRING",
          "mode": "NULLABLE"
        },
        {
          "name": "location",
          "type": "STRING",
          "mode": "NULLABLE"
        },
        {
          "name": "debug_info",
          "type": "STRING",
          "mode": "NULLABLE"
        },
        {
          "name": "message",
          "type": "STRING",
          "mode": "NULLABLE"
        }
      ]
    },
    {
      "name": "job_id",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "job_type",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "parent_job_id",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "priority",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "project_id",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "project_number",
      "type": "INTEGER",
      "mode": "NULLABLE"
    },
    {
      "name": "referenced_tables",
      "type": "RECORD",
      "mode": "REPEATED",
      "fields": [
        {
          "name": "project_id",
          "type": "STRING",
          "mode": "NULLABLE"
        },
        {
          "name": "dataset_id",
          "type": "STRING",
          "mode": "NULLABLE"
        },
        {
          "name": "table_id",
          "type": "STRING",
          "mode": "NULLABLE"
        }
      ]
    },
    {
      "name": "reservation_id",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "start_time",
      "type": "TIMESTAMP",
      "mode": "NULLABLE"
    },
    {
      "name": "state",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "statement_type",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "timeline",
      "type": "RECORD",
      "mode": "REPEATED",
      "fields": [
        {
          "name": "elapsed_ms",
          "type": "INTEGER",
          "mode": "NULLABLE"
        },
        {
          "name": "total_slot_ms",
          "type": "INTEGER",
          "mode": "NULLABLE"
        },
        {
          "name": "pending_units",
          "type": "INTEGER",
          "mode": "NULLABLE"
        },
        {
          "name": "completed_units",
          "type": "INTEGER",
          "mode": "NULLABLE"
        },
        {
          "name": "active_units",
          "type": "INTEGER",
          "mode": "NULLABLE"
        },
        {
          "name": "estimated_runnable_units",
          "type": "INTEGER",
          "mode": "NULLABLE"
        }
      ]
    },
    {
      "name": "total_bytes_billed",
      "type": "INTEGER",
      "mode": "NULLABLE"
    },
    {
      "name": "total_bytes_processed",
      "type": "INTEGER",
      "mode": "NULLABLE"
    },
    {
      "name": "total_modified_partitions",
      "type": "INTEGER",
      "mode": "NULLABLE"
    },
    {
      "name": "total_slot_ms",
      "type": "INTEGER",
      "mode": "NULLABLE"
    },
    {
      "name": "transaction_id",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "user_email",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "query_info_resource_warning",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "query_info_query_hashes_normalized_literals",
      "type": "STRING",
      "mode": "NULLABLE"
    },
    {
      "name": "transferred_bytes",
      "type": "INTEGER",
      "mode": "NULLABLE"
    },
    {
      "name": "creation_date",
      "type": "DATE",
      "mode": "NULLABLE"
    }
  ]
}
```

There are certainly type differences such as INTEGER for `estimated_runnable_units` here.